nfp/preprocessing/xtb_preprocessor.py

```diff
@@ -113,6 +135,9 @@ def get_node_features(
             node_features["atom_xtb"][n] = atom_dict["atom_xtb"]
         return node_features
 
+    def get_graph_features(self, graph_data: dict) -> Dict[str, np.ndarray]:
+        return {"mol_xtb": graph_data["mol_xtb"]}
```

Comment:
might want to make this 
```python
return {"mol_xtb": np.asarray(graph_data["mol_xtb"])}
```

or it could go above where you create the graph_data object